ios/fastlane/Fastfile

```diff
@@ -198,6 +198,56 @@ platform :ios do
     )
   end
 
+  desc "Submit a new Beta Build to TestFlight, using CircleCI"
+  lane :beta_circleci_testflight do |options|
+
+    # Setup the circleci environment
+    setup_ci(
+      provider: "circleci"
+    )
+
+    # use the App Store API Key to authenticate
+    api_key = app_store_connect_api_key(
+      key_id: "#{ENV['APP_STORE_API_KEY_ID']}",
+      issuer_id: "#{ENV['APP_STORE_API_KEY_ISSUER_ID']}",
+      key_filepath: "./fastlane/AuthKey_#{ENV['APP_STORE_API_KEY_ID']}.p8",
+      duration: 1200,
+    )
+
+    sync_code_signing(type: "appstore", api_key: api_key)
+
+    # Install pods
+    cocoapods
+
+    # build the app
+    gym(
+      scheme: "ItaliaApp",
+      clean: true,
+      export_method: "app-store",
+      workspace: "ItaliaApp.xcworkspace",
+      include_symbols: false,
+      # sdk: "iphoneos8.4"
```

Comment:
This can be removed?

Reply:
Yes, is just an old comment, removed in 0552807de8cdcab2181e6ba7a75603c231a1bee2